src/ansys/aedt/core/modeler/cad/polylines.py

```diff
@@ -615,6 +615,7 @@ def _point_segment_string_array(self):
                     if current_segment.type == "AngularArc":
                         points_str.append(self._pl_point(current_segment.extra_points[i - 1]))
                         index_count += 1
+                        pos_count_incr += 1
```

Comment:
Can you add a test to show why previous implementation should have been failing?

Reply:
sure!